ledfx/nowplaying/providers/smtc.py

```
@@ -167,16 +384,219 @@ async def _fetch_and_push_metadata(self):
                 artist=artist,
                 album=album,
+                position=position,
+                duration=duration,
+                playing=playing,
+            ),
+            has_own_artwork=True,
+        )
+        _LOGGER.debug("SMTC: metadata pushed for %s", title)
+
+        # Fire-and-forget: artwork arrives when it arrives.
+        asyncio.ensure_future(self._push_artwork(props))
```

Ruff (0.16.1)

[warning] Store a reference to the return value of `asyncio.ensure_future`

(RUF006)